Trademark guidelines update (#4030)
jenkins-infra/jenkins.io: content/project/trademark/index.adoc

+
[quote]
____

> Jenkins(R) is a registered trademark of the Linux Foundation.

Author: ```suggestion
Jenkins(R) is a registered trademark of LF Charities Inc.
```

[quote]
____
The registered trademark Jenkins(R) is used pursuant to a sublicense from the

> Jenkins project and the Linux Foundation.

Author: ```suggestion
Jenkins project and the LF Charities Inc.
```

> Jenkins project and the LF Charities Inc.

Reviewer: ```suggestion
Jenkins project and LF Charities Inc.
```

* https://www.linuxfoundation.org/en/trademarks/[Linux Foundation Trademarks]
* https://www.linuxfoundation.org/trademark-usage/[Linux Foundation Trademark Usage Guidelines]
* link:approved-usage[Approved Trademark Usage (before Feb 24, 2021)]
* link:sublicense[Trademark Sublicense Agreement]

== Change History

=== Feb 24, 2021

The Jenkins Trademark was officially transferred from 
link:https://spi-inc.org[Software in the Public Interest, Inc.] 
to the Linux Foundation (its "LF Charities, Inc." subsidiary).
link:https://www.linuxfoundation.org/trademark-usage/[Linux Foundation trademark usage guidelines]
are now strictly followed by the project.

=== Oct 14, 2020

The link:/project/governance-meeting[Jenkins Governance Meeting] reviewed the trademark usage policy
and voted for closer alignment with the link:https://www.linuxfoundation.org/trademark-usage/[trademark usage guidelines] defined by The Linux Foundation.

* We adopt Linux Foundation pre-approved trademark patterns and usage guidelines.
  As long as the naming patterns are followed, no approval from the project is needed.

> * We are not applying requirements retrospectively, all link:approved-usage[previously approved trademark usages] remains approved.

Reviewer 2: I believe the alternate word better fits this case.
```suggestion
* We are not applying requirements retroactively, all link:approved-usage[previously approved trademark usages] remains approved.
```

> * Until the transition to CDF is over, we may make exceptions in the naming policy.
>   Example: precedent in already approved trademarks.

Author: ```suggestion
* Until the transition to CDF is over, we may make exceptions in the naming policy,
  subject to approval at the Jenkins Governance Meeting.
```

reasonable facsimile of these attributions may be used. In case of doubt as to
the proper shortened form, examples may be submitted to the Jenkins project for
approval.

== References

* link:https://trademarks.justia.com/854/47/jenkins-85447465.html[Jenkins - Trademark Details #4664929]
* link:https://www.linuxfoundation.org/trademark-usage/[trademark usage guidelines] by The Linux Foundation
* link:approved-usage[Approved Trademark Usage]
* link:sublicense[Trademark Sublicense Agreement]

== Change History

=== Oct 14, 2020

The link:/project/governance-meeting[Jenkins Governance Meeting] reviewed the trademark usage policy
and voted for closer alignment with the link:https://www.linuxfoundation.org/trademark-usage/[trademark usage guidelines] defined by The Linux Foundation.

* We adopt Linux Foundation pre-approved trademark patterns and usage guidelines.
  As long as the naming patterns are followed, no approval from the project is needed.
* We are not applying requirements retrospectively, all link:approved-usage[previously approved trademark usages] remains approved.
* Until the transition to CDF is over, we may make exceptions in the naming policy.

>   Example: precedent in already approved trademarks.

Reviewer: It's not precedent if it followed the rules at the time.

We're just grandfathering in previous approvals, RHEL style (IIUC).

In fact, there is currently no approved trademark usage since we started moving to CDF/LF that would be considered an exception.
